### DISCUSSION TITLE
Localization strings are not shown on WPF UI controls

### DISCUSSION THREAD

**Discussion by tshor-omegatmm** [SeriousM/WPFLocalizationExtension#389] [Q&A] [answered]
Hi,
I have wpf app (dotnet 6.0), using this extension. I am unable to get the translated strings from the Resources.*.resx during runtime. Design Time - I can change DesignCulture to another culture and ui changes accordingly. Also, when I use localized strings by calling it in the code : "Properties.Resources.MyTranslatedString" it works like a charm. English culture works fine in runtime (all the strings elements using: 
`Text="{lex:Loc MyTranslated_String}"`

I have gone through all the Q&A and my friend Google :-), but i'm still stuck. What could I be missing?

Anyone had this issue?
Your help is appreciated.

Tim

**Comment by Karnah** [comment, marked as answer]
Hello, @kazachok13!
Could you provide a small reproduce repo? I've tried move [HelloWorldWPF](https://github.com/XAMLMarkupExtensions/WPFLocalizeExtension/tree/master/tests/HelloWorldWPF) to net6.0 and it worked fine.

Also, please check, how you change culture. Do you use ```LocalizeDictionary.Instance.Culture = new System.Globalization.CultureInfo("xxx")```?

**Reply by tshor-omegatmm** [reply]
Thank you,
the LocalizeDictionary.Instance.Culture set is what it needed. Weirdly enough, we used this plug in in our previous version and resource files were non Public and we never user the LocalizeDicitionary.Instance.Culture = new(), and it worked. :-()

Thanks
